### DISCUSSION TITLE
Have a region extend outside of wavesurfer?

### DISCUSSION THREAD

**Discussion by Lowgy** [katspaugh/wavesurfer.js#3977] [Q&A] [answered]
Is it possible to have a region position absolute "outside" of the waveform? 
My goal is to have a special region I can create that extends outside of the waveform to allow of dragging / resizing from "outside" the waveform container.

Basically just have it extend 10-20px out the top and bottom of the waveform

**Comment by katspaugh** [comment, marked as answer]
You can do it with some custom CSS:

https://wavesurfer.xyz/examples/?acf48e77afc43ea91be2dd93eae98721

**Reply by Lowgy** [reply]
You are the literal goat! Thank you!

**Reply by Lowgy** [reply]
Is there a link to that example in the docs? I swear I searched high and low through it

**Reply by katspaugh** [reply]
It's a new example I made for this answer.
